store/modules/advocates.ts

```diff
@@ -35,28 +41,48 @@ export {
   Advocate
 }
 
-export default {
-  state () {
-    return {
-      advocates: []
-    }
+export class State {
+  advocates: Advocate[] = []
+  regionFilters: string[] = []
+}
+
+const actions = <ActionTree<State, any>> {
```

Comment:
As pointed out offline, the Vuex documentation also orders them 1. Mutations, 2. Actions, 3. Getters: https://vuex.vuejs.org/guide/modules.html#modules

Despite this, I agree that 1. Getters, 2. Mutations, 3. Actions makes probably more sense, due to _getters_ usually being first on classes, and _actions_ referencing existing _mutations_.

We should write this convention down for future reference. It could maybe be included as part of #1030.

Reply:
I'll move the order as suggested 🙂